android/app/src/main/java/io/imagekit/imagekitdemo/UploadAuthViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package io.imagekit.imagekitdemo
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class UploadAuthViewModel : ViewModel() {
+    private val service: UploadAuthService = Retrofit.Builder()
+        .baseUrl("AUTH_SERVER_URL")
```

Comment:
We should move these to a `constants.kt` file and use the values everywhere we need

Reply:
Done. Please check